test/tools/runner/index.js

```diff
@@ -86,7 +86,7 @@ before(function(_done) {
         }
 
         this.configuration = new TestConfiguration(parsedURI, context);
-        done();
+        client.close(done);
```

Comment:
I've fallen for this before myself :) We _do_ call `close` inside of `done` on L65

Reply:
Whoops! thanks.